libs/components/tabs/src/lib/modules/tabs/tab-button.component.ts

```diff
@@ -83,4 +145,12 @@ export class SkyTabButtonComponent {
     event.stopPropagation();
     event.preventDefault();
   }
+
+  public focusBtn(): void {
+    this.#adapterService.focusButtonLink(this.#elementRef);
+  }
+
+  public onFocus(): void {
+    this.#tabsetService.setFocusedTabBtnIndex(this.tabIndex);
```

Comment:
This covers our bases and updates the focused tab index if focus is received from something _other_ than an arrow/home/end key (i.e.a click,  tab, shift + tab). It ensures that when they use the arrow keys again, we start from this tab button instead of the last one that received focus from an arrow key.